src/Tizen.NUI/src/public/BaseComponents/TextField.cs

```diff
@@ -1854,9 +1854,10 @@ private bool InternalEnableEditing
         }
 
         /// <summary>
+        /// PrimaryCursorPosition property.<br />
         /// Specify primary cursor (caret) position in text control.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 10 </since_tizen>
         public int PrimaryCursorPosition
         {
```

Comment:
both TextEditor.PrimaryCursorPosition comments apply here too

Reply:
@wiertel , The suggestion has been applied. Could you check it , please.